modules/candidate_parameters/ajax/getData.php

```diff
@@ -470,3 +472,29 @@ function getConsentStatusHistory($pscid)
     }
     return $formattedHistory;
 }
+
+/**
+ * Handles the fetching of candidate's date of birth.
+ *
+ * @throws DatabaseException
+ *
+ * @return array
+ */
+function getDOBFields()
+{
+    $candID = $_GET['candID'];
+    $db     = \Database::singleton();
+    // Get PSCID
+    $candidateData = $db->pselectRow(
+        'SELECT * FROM candidate where CandID =:candid',
```

Comment:
```suggestion
        'SELECT PSCID,DoB FROM candidate where CandID =:candid',
```

Might as well only ask for what you need.